config/api.yml

```diff
@@ -428,8 +428,11 @@
     :description: Reports
     :options:
     - :collection
-    :methods: *70174834086080
+    :methods: *70174834085860
     :klass: MiqReport
+    :resource_actions:
+      :post:
+      - :name: run
```

Comment:
We need a role identifier here, otherwise anyone can run reports. most likely "miq_report_run" as per db/fixtures/miq_product_features.yml